README.md

```diff
@@ -13,60 +9,24 @@ OpenCL bindings for Julia
 [buildbot-travis-img]: https://travis-ci.org/JuliaGPU/OpenCL.jl.svg?branch=master
 [buildbot-travis-url]: https://travis-ci.org/JuliaGPU/OpenCL.jl?branch=master
 
-Julia interface for the OpenCL parallel computation API
-
-This package aims to be a complete solution for OpenCL programming in Julia, similar in scope to [PyOpenCL] for Python.
-It provides a high level api for OpenCL to make programing GPU's and multicore CPU's much less onerous.
-
-OpenCL.jl provides access to OpenCL API versions 1.0, 1.1, 1.2 and 2.0.
+*Julia interface for the OpenCL parallel computation API*
 
-#### This package is based off the work of others:
-  * [PyOpenCL] by Andreas Klockner
-  * [oclpb]    by Sean Ross
-  * [Boost.Compute] by Kyle Lutz
-  * [rust-opencl]
-
-[PyOpenCL]: http://mathema.tician.de/software/pyopencl/
-[oclpb]: https://github.com/srossross/oclpb
-[Boost.Compute]:https://github.com/kylelutz/compute
-[rust-opencl]: https://github.com/luqmana/rust-opencl
+This package aims to be a complete solution for OpenCL programming in Julia, similar in scope to [PyOpenCL] for Python. It provides a high level API for OpenCL to make programing hardware accelerators, such as GPUs, FPGAs, and DSPs, as well as multicore CPUs much less onerous.
 
-OpenCL.jl has had contributions from [many developers](https://github.com/JuliaGPU/OpenCL.jl/graphs/contributors).
+OpenCL.jl provides access to [OpenCL API](https://www.khronos.org/registry/OpenCL/) versions 1.0, 1.1, 1.2 and 2.0.
 
-## Currently supported Julia versions
-- Julia `v"0.4.x"` is supported on the `release-0.4` branch and the OpenCL.jl versions `v"0.4.x"`. Only bug-fixes will be applied.
-- Julia `v"0.5.x"` is supported on the `master` branch and the OpenCL.jl versions `v"0.5.x"`.
-- Julia `v"0.6.x"` is *experimentally* supported on the `master` branch and the OpenCL.jl versions `v"0.5.x"`.
-
-### Discontinued support
-- Julia `v"0.3.x"` was supported on OpenCL.jl versions `v"0.3.x"`. It should still be installable and work.
-
-## Setup
-1. Install an OpenCL driver. If you use OSX, OpenCL is already available
-2. Checkout the packages from the Julia repl
-
-```julia
-  Pkg.add("OpenCL")
-```
-
-3. OpenCL will be installed in your ``.julia`` directory
-4. ``cd`` into your ``.julia`` directory to run the tests and try out the examples
-5. To update to the latest development version, from the Julia repl:
+## Installation
+1. Install an OpenCL driver. (If you're on macOS, OpenCL should already be available.)
```

Comment:
We should specify what versions of macOS are supported (since OpenCL support went away in macOS recently).

Reply:
Hm, I'm on macOS Big Sur and and it works, but maybe I installed it with some other software. Anyways, we could just drop the macOS comment entirely to be safe. What do you think?

Reply:
We should probably say instead, "If you're on macOS, OpenCL is either already available or unsupported"?